src/envo/envo-edit.owl

```diff
@@ -44939,6 +44944,36 @@ AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/ENVO_03605000>
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03605000> "periphytic biofilm"@en)
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03605000> <http://purl.obolibrary.org/obo/ENVO_01000156>)
 
+# Class: <http://purl.obolibrary.org/obo/ENVO_03605001> (epilithon)
+
+AnnotationAssertion(Annotation(<http://purl.obolibrary.org/obo/IAO_0000119> "NEON.DOC.003044, revision E, title AOS Protocol and Procedure: AMC – Aquatic Microbial Sampling in https://data.neonscience.org/documents/10179/1883159/NEON.DOC.003044vE/05ca8fcc-33ba-a5b2-0e93-8f4001f38c37") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03605001> "Periphyton colonizing rock substrata")
```

Comment:
this isn't how we do definition sources in the rest of envo

Reply:
> this isn't how we do definition sources in the rest of envo

Isn't that what I did?

![image](https://github.com/EnvironmentOntology/envo/assets/25185273/5b3ae50d-1b82-4d2d-b52d-6429a346c728)